### DISCUSSION TITLE
*Need 7 more stars for my Starstruck badge! 🌟*

### DISCUSSION THREAD

**Discussion by mooosakhan** [community/community#147800] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

**Need 7 more stars for my Starstruck badge! 🌟**

Hey everyone! I'm just a few stars away from unlocking the prestigious *Starstruck* badge, and I need *7 more* stars to reach the goal. If you love the project or just want to help out, consider giving it a star! Your support means the world to me, and it helps bring more visibility to the project. 

As a thank you, I'll be sharing a special update once I hit the target! 💥

Let's make this happen! ⭐

**Comment by shinybrightstar** [comment]
Hi @mooosakhan , we’re glad you’re here 😀

Thanks for sharing your project with the community. Unfortunately, we currently do not allow self-promotion or advertising on the Community Discussions. We want to make sure there is space for users to ask questions without overwhelming them with other conversations. Thank you for helping us maintain a productive and tidy community for all our members.

Good luck with getting the rest of your stars!
